tests_python/test_debugger_json.py

```diff
@@ -5739,26 +5739,6 @@ def test_stop_on_entry2(case_setup_dap):
         writer.finished_ok = True
 
 
-def test_stop_on_entry_verify_strings(case_setup_dap):
```

Comment:
This test doesn't work anymore since there's no longer a special case for pydevd strings.